docs/resources/cdm_link.md

```diff
@@ -0,0 +1,145 @@
+---
+subcategory: "Cloud Data Migration (CDM)"
+---
+
+# huaweicloud_cdm_link
+
+Manages a link resource within HuaweiCloud. A link enables the CDM cluster to read data from and write data to
+ a data source.
+
+## Example Usage
+
+### Link to OBS
+
+```hcl
+variable "obs_name" {}
+variable "obs_link_name" {}
+variable "cdm_cluster_id" {}
+variable "access_key" {}
+variable "security_key" {}
+
+resource "huaweicloud_obs_bucket" "bucket" {
+  bucket        = var.obs_name
+  acl           = "private"
+  force_destroy = true
+}
+
+resource "huaweicloud_cdm_link" "obsLink" {
+  name       = var.obs_link_name
+  connector  = "obs-connector"
+  cluster_id = var.cdm_cluster_id
+
+  config = {
+    "storageType" = "OBS"
+    "server"      = trimprefix(huaweicloud_obs_bucket.bucket.bucket_domain_name, "${huaweicloud_obs_bucket.bucket.bucket}.")
```

Comment:
can the function of removing the prefix be handled in the code?

Reply:
no, According to different data sources, the config is different.  detail is listed in document